pkg/app/app.go

```diff
@@ -89,6 +90,15 @@ func defaultFuncApiUrl(_ configuration.Configuration, logger *zerolog.Logger) co
 
 		if len(urlFromOauthToken) > 0 && len(urlFromOauthToken[0]) > 0 {
 			urlString = urlFromOauthToken[0]
+		} else if auth.IsAuthTypePAT(config.GetString(configuration.AUTHENTICATION_TOKEN)) {
```

Comment:
Suggestion: maybe avoid the duplication of `config.GetString(configuration.AUTHENTICATION_TOKEN)`